apps/endorsement-system/src/app/modules/endorsement/e2e/endorsement.spec.ts

```diff
@@ -77,9 +77,22 @@ describe('Endorsement', () => {
       statusCode: 404,
     })
   })
+  it(`POST /endorsement-list/:listId/endorsement should fail to create endorsement on a closed list`, async () => {
+    const response = await request(app.getHttpServer())
+      .post(
+        '/endorsement-list/9c0b4106-4213-43be-a6b2-ff324f4ba0c3/endorsement',
```

Comment:
I see the ids ending with `ba0c3` here but elsewhere it is ending with `ba0c1`.
Are these unrelated?

Reply:
Yes , in the seeder we import three endorsements ending in `ba0c1`, `ba0c2`, `ba0c3`.
Seeder is run before running e2e and tests empty the db after ensuring this data is always the same

Reply:
Just making sure